src/stores/Comment/CommentStore.js

```diff
@@ -164,7 +164,8 @@ export const CommentStore = types
 
       if (self.annotationId) {
         comment.annotation = self.annotationId;
-      } else if (self.draftId) {
+      }
+      if (self.draftId) {
```

Comment:
the logic here: if there is already submitted annotation, we should not attach comment to a draft, only to annotation directly.

Reply:
but LSF is actually a standalone separated lib actually, so maybe it's better to do it like in your fix and move this conditional logic to api part (dm)

Reply:
oh, I see, in both places it sends correct ids to api!